source/agora/cli/SendTxProcess.d

```diff
@@ -0,0 +1,329 @@
+    outputs ~= "";
+    outputs ~= "   sendtx      Send a transaction to node";
+    outputs ~= "";
+    outputs ~= "        -i --ip      IP address of node";
+    outputs ~= "        -p --port    Port of node";
+    outputs ~= "        -t --txhash  Hash of the previous transaction that";
+    outputs ~= "                     contains the Output which the new ";
+    outputs ~= "                     transaction will spend";
+    outputs ~= "        -n --index   The index of the output in the previous";
+    outputs ~= "                     transaction which will be spent";
+    outputs ~= "        -a --amount  The amount to spend";
+    outputs ~= "        -d --dest    The address key to send the output";
+    outputs ~= "        -k --key     The seed used to sign the new transaction";
+    outputs ~= "        -o --dump    Dump output option";
+    outputs ~= "";
+}
+
+/*******************************************************************************
+
+    Input an arguments, generate the transaction and send it to the node
+
+    Params:
+        args = Cli command line arguments
+
+*******************************************************************************/
+
+public int sendTxProcess (string[] args, ref string[] outputs,
+                            APIMaker api_maker)
+{
+    SendTxOption op;
+    GetoptResult res;
+
+    try
+    {
+        res = parseSendTxOption(op, args);
+        if (res.helpWanted)
+        {
+            printSendTxHelp(outputs);
+            return CLI_SUCCESS;
+        }
+    }
+    catch (Exception ex)
+    {
+        printSendTxHelp(outputs);
+        return CLI_EXCEPTION;
+    }
+
+    bool isValid = true;
+
+    if (op.host == "")
+        op.host = "localhost";
+
+    if (op.port == 0)
+        op.port = 2826;
+
+    if (op.txhash.length == 0)
+    {
+        if (isValid) printSendTxHelp(outputs);
+        outputs ~= "Previous Transaction hash is not entered.[--txhash]";
+        isValid = false;
+    }
+
+    if (op.amount == 0)
+    {
+        if (isValid) printSendTxHelp(outputs);
+        outputs ~= "Amount is not entered.[--amount]";
+        isValid = false;
+    }
+
+    if (op.address.length == 0)
+    {
+        if (isValid) printSendTxHelp(outputs);
+        outputs ~= "Address is not entered.[--dest]";
+        isValid = false;
+    }
+
+    if (op.key.length == 0)
+    {
+        if (isValid) printSendTxHelp(outputs);
+        outputs ~= "Key is not entered.[--key]";
+        isValid = false;
+    }
+
+    if (!isValid)
+        return CLI_INVALIDE_AGUMENTS;
+
+    // create the transaction
+    auto key_pair = KeyPair.fromSeed(Seed.fromString(op.key));
+
+    Transaction tx =
+    {
+        [Input(Hash.fromString(op.txhash), op.index)],
+        [Output(Amount(op.amount), PublicKey.fromString(op.address))]
+    };
+
+    auto signature = key_pair.secret.sign(hashFull(tx)[]);
+    tx.inputs[0].signature = signature;
+
+    if (op.dump)
```

Comment:
if `dump` is enabled, the tool should not send the transaction. Dumping is used to inspect the transaction so it's correct before it's being sent.

So add a `return CLI_SUCCESS` here after the outputs are written to.